aws-lambda-java-runtime-interface-client/pom.xml

```diff
@@ -265,6 +265,28 @@
                     </execution>
                 </executions>
             </plugin>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
Latest version seems to be 1.2.2, could you please update it?
https://mvnrepository.com/artifact/org.moditect/moditect-maven-plugin/1.2.2.Final